google-login-plugin/src/com/google/gct/login/stats/GctConfigurable.java

```diff
@@ -0,0 +1,127 @@
+package com.google.gct.login.stats;
+
+import com.intellij.ide.BrowserUtil;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.openapi.options.SearchableConfigurable;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import javax.swing.border.EtchedBorder;
+import javax.swing.border.TitledBorder;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.net.URL;
+
+/**
+ * Implementation of {@code applicationConfigurable} extension that provides a
+ * Google Cloud Tools tab in the "Settings" dialog.
+ */
+public class GctConfigurable implements SearchableConfigurable {
+    private JCheckBox enableUsageTrackerBox;
+    private static final Logger LOG = Logger.getInstance(GctConfigurable.class);
+    private static final String ENABLE_TRACKER_TEXT =
+            "Would you like to help Google improve the Cloud Tools for IntelliJ plug-in by sharing your usage data?";
+    private static final String PRIVACY_POLICY_URL = "http://www.google.com/policies/privacy/";
+    private static final String PRIVACY_POLICY_TEXT = "<html>\nYour use of this plugin is subject to "
+            + "the Apache 2.0 software license and the <a href=\"" + PRIVACY_POLICY_URL
+            + "\">Google Privacy Policy</a>.</html>";
+
+
+    public GctConfigurable() {
+    }
+
+    @NotNull
+    @Override
+    public String getId() {
+        return "google.usage.tracker";
+    }
+
+    @Nullable
+    @Override
+    public Runnable enableSearch(String option) {
+        return null;
+    }
+
+    @Nls
+    @Override
+    public String getDisplayName() {
```

Comment:
Yes, but the login plugin is only going to be shipped as part of the Cloud Tools plugin. To my knowledge, the usage tracking functionality is in the login plugin because of a dependency issue and not because the usage tracking and logging are tied together. @loosebazooka